frameworks/CSharp/beetlex/PlatformBenchmarks/DBRaw.cs

```diff
@@ -161,7 +162,7 @@ private async Task<World[]> LoadMultipleRows(int count, DbConnection db)
 
         public async Task<List<Fortune>> LoadFortunesRows()
         {
-            var result = new List<Fortune>(20);
+            var result = new List<Fortune>(ListDefaultSize);
```

Comment:
Again, this is not acceptable. Please remove this. You can fix the other two you pointed out in this PR if you'd like, or I can get to them tomorrow. It was missed.

> The list data structure must be a dynamic-size or equivalent and should not be dimensioned using foreknowledge of the row-count of the database table.

Reply:
Can I use Array buffer, dynamically-sized returned ArraySegment?